src/nccl_ofi_rdma.c

```diff
@@ -2689,12 +2714,12 @@ static int dereg_mr_ep(nccl_net_ofi_rdma_mr_handle_t *mr_handle,
 	int ret = 0;
 
 	if (OFI_UNLIKELY(mr_handle == NULL)) {
-		NCCL_OFI_TRACE(NCCL_INIT | NCCL_NET, "Null MR handle provided. This is an error.");
+		NCCL_OFI_WARN("Null MR handle provided. This is an error.");
 		return ncclInternalError;
 	}
 
-	if (OFI_UNLIKELY(mr_handle->num_rails < 1)) {
-		NCCL_OFI_TRACE(NCCL_INIT | NCCL_NET, "Unexpected number of rails in rdma memory registration handle");
+	if (OFI_UNLIKELY(mr_handle->num_rails < 0)) {
```

Comment:
because in the initial handshake, you have 0 rails setup (the control channel is not a rail, because it can't send data).  I'm not actually convinced this test is still useful, outside of bogus value checking.